apps/api/src/database/migrations/20230601144617_remove_end_date_column_examination_items_table/migration.sql

```diff
@@ -0,0 +1,25 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `endDate` on the `ExaminationTimetableItems` table. All the data in the column will be lost.
+
+*/
+BEGIN TRY
+
+BEGIN TRAN;
+
+-- AlterTable
+ALTER TABLE [dbo].[ExaminationTimetableItems] DROP COLUMN [endDate];
```

Comment:
for consistency, should this be singular, rather than plural ie - ExaminationTimetableItem ?  All our other tables are singular - eg Case, Document etc.

Reply:
I know it means a few name updates throughout, but it would give expected consistency, so I think it would be worth it!